src/components/App/App.vue

```diff
@@ -161,63 +171,55 @@ export default {
-        .find(item => item.key === 'change_role_task:complete_auto_verification')
-        .uint32_value
-      config.CHANGE_ROLE_TASKS.submitAutoVerification = tasks
-        .find(item => item.key === 'change_role_task:submit_auto_verification')
-        .uint32_value
-      config.CHANGE_ROLE_TASKS.manualReviewRequired = tasks
-        .find(item => item.key === 'change_role_task:manual_review_required')
-        .uint32_value
+      config.CHANGE_ROLE_TASKS.completeAutoVerification = this.kvEntries
+        .find(item => item.id === 'change_role_task:complete_auto_verification')
+        .value.u32
+      config.CHANGE_ROLE_TASKS.submitAutoVerification = this.kvEntries
+        .find(item => item.id === 'change_role_task:submit_auto_verification')
+        .value.u32
+      config.CHANGE_ROLE_TASKS.manualReviewRequired = this.kvEntries
+        .find(item => item.id === 'change_role_task:manual_review_required')
+        .value.u32
     },
 
     async loadAccountRolesConfigs () {
-      const { body: roles } =
-        await this.$http.get(`${config.HORIZON_SERVER}/key_value`)
-      config.ACCOUNT_ROLES.notVerified = roles
-        .find(item => item.key === 'account_role:unverified')
-        .uint32_value
-      config.ACCOUNT_ROLES.general = roles
-        .find(item => item.key === 'account_role:general')
-        .uint32_value
-      config.ACCOUNT_ROLES.usAccredited = roles
-        .find(item => item.key === 'account_role:us_accredited')
-        .uint32_value
-      config.ACCOUNT_ROLES.usVerified = roles
-        .find(item => item.key === 'account_role:us_verified')
-        .uint32_value
-      config.ACCOUNT_ROLES.corporate = roles
-        .find(item => item.key === 'account_role:corporate')
-        .uint32_value
-      config.ACCOUNT_ROLES.blocked = roles
-        .find(item => item.key === 'account_role:blocked')
-        .uint32_value
-      config.SIGNER_ROLES.default = roles
-        .find(item => item.key === 'signer_role:default')
-        .uint32_value
+      config.ACCOUNT_ROLES.notVerified = this.kvEntries
+        .find(item => item.id === 'account_role:unverified')
+        .value.u32
+      config.ACCOUNT_ROLES.general = this.kvEntries
+        .find(item => item.id === 'account_role:general')
+        .value.u32
+      config.ACCOUNT_ROLES.usAccredited = this.kvEntries
+        .find(item => item.id === 'account_role:us_accredited')
+        .value.u32
+      config.ACCOUNT_ROLES.usVerified = this.kvEntries
+        .find(item => item.id === 'account_role:us_verified')
+        .value.u32
+      config.ACCOUNT_ROLES.corporate = this.kvEntries
+        .find(item => item.id === 'account_role:corporate')
+        .value.u32
+      config.ACCOUNT_ROLES.blocked = this.kvEntries
+        .find(item => item.id === 'account_role:blocked')
+        .value.u32
+      config.SIGNER_ROLES.default = this.kvEntries
+        .find(item => item.id === 'signer_role:default')
+        .value.u32
```

Comment:
the main purpose of the task is to remove the key values from `config.js`, cuz `config.js` should contain only arguments to configure the instance. key-values are set via admin and cannot be customized, so should not be placed in configuration file

so pls replace usages of the key values from `config` with usages them from `store`